files/grepper.go

```diff
@@ -51,30 +51,29 @@ func GWalker(path string, f os.FileInfo, err error) error {
 
 	// only try to open existing files
 	if _, err := os.Stat(path); !os.IsNotExist(err) && !f.IsDir() {
-		isImage, err := IsImage(path)
+		err := IsImage(path)
 		if err != nil {
 			log.Fatal(err)
```

Comment:
This isn't good, `log.Fatal(err)` will kill the program is a non-image is encountered. This err could be an `ErrImageNotRecognized` error, which should indicate that the program skip processing this file, instead of dying :(